Bottom: fc5beae6ce707ddd4309d55f3a60bb459cc61785
Top:    1e6942b631d5e392b182fb4fa558d12b5a8a660f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 16:30:04 +0000

Refresh of attempt-to-refactor-the-step

---

diff --git a/src/hid/step/object3d_step.c b/src/hid/step/object3d_step.c
index 3464ff2..573a12c 100644
--- a/src/hid/step/object3d_step.c
+++ b/src/hid/step/object3d_step.c
@@ -146,7 +146,7 @@ object3d_export_to_step (object3d *object, const char *filename)
       face->surface_identifier =
         step_plane (step, "NONE",
                     step_axis2_placement_3d (step, "NONE",
-                                             step_cartesian_point (step, "NONE", ov->x,  /* A point on the plane. Defines 0,0 of the plane's parameterised coords. */ 
+                                             step_cartesian_point (step, "NONE", ov->x,  /* A point on the plane. Defines 0,0 of the plane's parameterised coords. */
                                                                                  ov->y,      /* Set this to the origin vertex of the first edge */
                                                                                  ov->z),     /* this contour links to in the quad edge structure. */
                                                    step_direction (step, "NONE", face->nx, face->ny, face->nz), /* An axis direction normal to the the face - Gives z-axis */
